### DISCUSSION TITLE
Vote on new Tycho logo!

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#2305] [Polls]
## Hi Tycho Users,

in the past weeks we have been working with a design agency to get a new logo for Tycho (see https://github.com/eclipse-tycho/tycho/discussions/1633) and finally have identified three candidates that we think would fit, to make a final dessision (and probably making minor adjustments), we like to get some feedback from you.

## The Logos

![grafik](https://user-images.githubusercontent.com/1331477/225841746-7a585acd-1712-416d-a48e-692b9e428010.png)

## Some background that leads to the logo design

1. Based on the general shape of the Eclipse Logo and using the "Eclipse Colors" this shows the connection of Tycho with its connection of Tycho and Eclipse, the tree dots symbolize different assets produced from a build forming some planets orbiting the center (Tycho/Maven) that connects them.
2. Similar to the first, this is more elementary and reduced to give a more abstract view, this also has the appearance of an `O` what can symbolize a the cycle in a build, or in a continuous development. 
3. The hexagon symbolize stability and reliability as Tycho connecting Maven with PDE world on  different edges / aspects from development to deployment, with a homage to a solar eclipse of the foundation logo.

## Your feedback is appreciated!

Just cast a vote what logo you think would best fit what Tycho symbolize for you! We also would appreciate any comments or suggestions!

**Comment by kjsmita6** [comment]
Some initial thoughts:

The 1st one isn't exactly the "Eclipse color", as the Eclipse icon on my computer has RGB(243, 129, 25) while that one is RGB(229, 106, 30), which looks slightly darker than what Eclipse's is. In contrast, the 2nd one is lighter than the Eclipse color. Overall, with my initial glance at all three icons on the same page, the 2nd looks more like Eclipse's color palette and style than the others (even if it isn't).

Related, the 2nd icon appears flat, while the 1st icon appears to be a sphere (because of the lighting), which again matches Eclipse's style and slightly makes more sense to me than a 3D logo.

Is there a reason for the 3rd one why the icon is on the left of Tycho™ instead of the right? It would be nice to include that in the "story" if possible (if there is one).

But they are very nice icons overall! Glad to see Tycho getting a logo :)

**Reply by laeubi** [reply]
@kjsmita6 thanks for your feedback, of course we can do smaller adjustments to the logo as well later on, for now its more to get a decision, I think especially things like right/left should be easy to change but I assume it is mostly because of the usual "right-to-left" reading where the logo comes first and then the text (as for example for github that shows your image first and then the name) that makes it most usual.

Also I'd like to note that the idea is that the logo could also be used without the text (if appropriate).

So as of today I see we have a quite strong preference on the third option (50%) and a split for the first and second option with a slight preference to the second (26 % versus 23%).

I noticed that I did not set an "end date" for the vote but hope that's fine as its also not an "official" vote :-)

I'll watch this now for a while if we get some more feedback but thin we will then go on with the third option!

**Comment by laeubi** [comment]
The logo was now contributed and in cooperated into the repository! :+1: 

- https://github.com/eclipse-tycho/tycho/pull/2366

**Reply by Bananeweizen** [reply]
Just wanted to mention that this looks really nice in the github UI (and probably all other places showing the logo now).
![image](https://user-images.githubusercontent.com/406876/236316791-369ba5c0-a715-4f90-80f0-3d30719739aa.png)

**Reply by laeubi** [reply]
Great you like it currently the logo is used on the [Tycho project page](https://projects.eclipse.org/projects/technology.tycho) and on the github org and social preview and the readme, it would be good to show it on [the documentation](https://tycho.eclipseprojects.io/doc/latest/index.html) as well, so if anyone likes to do so or has other ideas, feel free to open a PR :-)
